pb/dns.pb.go

```diff
@@ -18,7 +18,8 @@ import fmt "fmt"
 import math "math"
 
 import (
-	context "golang.org/x/net/context"
+	context "context"
```

Comment:
This is a generated file, we shouldn't be updating it. Does the latest version of `protoc-gen-go` use `context`?

Reply:
Hmm: https://github.com/golang/protobuf/blob/master/protoc-gen-go/grpc/grpc.go#L56